test/test_torch.py

```diff
@@ -1885,24 +1885,27 @@ def test_logical_not(self):
 
     @staticmethod
     def _test_logical_xor(self, device):
-        for dtype in (torch.bool,):  # Will add more dtypes in the future
+        for dtype in torch.testing.get_all_dtypes():
+            if dtype == torch.bfloat16:
+                continue
```

Comment:
nit: it's generally better to do an assertRaises here so that if we add support for bfloat16, we will be forced to update the test to ensure it's tested.

Otherwise we'll just never test this.

Reply:
@gchanan  Updated!